src/utils/objectFit/objectFit.test.ts

```diff
@@ -0,0 +1,37 @@
+import { contain, cover } from './objectFit.js';
+
+describe('objectFit', () => {
+  describe('contain', () => {
+    it('returns expected values for positive input arguments', () => {
+      expect(contain(100, 100, 50, 50)).toEqual({
+        x: 0,
+        y: 0,
+        width: 100,
+        height: 100,
+        scale: 2,
+        cssText: 'left:0px;top:0px;width:100px;height:100px;',
+      });
+    });
+
+    it('throws an error for non-positive input arguments', () => {
```

Comment:
Technically, `0` is positive.
Maybe reword (and also in other places) to "greater than 0" ?